capstone/scripts/convert_s3.py

```diff
@@ -213,6 +216,10 @@ def export_cases_by_volume(
 
 # Reporter-specific helper functions
 
+# Some reporters share a slug, so we have to differentiate with ids
+reporter_slug_dict = {"415":"us-ct-cl", "657":"wv-ct-cl",
```

Comment:
I think this won't work with string keys?

Reply:
I tested it and it does work with string keys, also backed up by how on line 79, `type(reporter_id) <class 'str'>`:

 https://github.com/harvard-lil/capstone/pull/2163/files#diff-b957af52fc247bd70b117433d1fb7fbdf64bdbff62ce5259934ffa1323f6c4d2R68

Reply:
oh I see, those IDs are from the API rather than database models -- carry on!